### DISCUSSION TITLE
Regenerate the cells after cutting

### DISCUSSION THREAD

**Discussion by cheny2024** [neperfepx/neper#910] [Q&A] [answered]
Can I get some help about generate a 3D model using Neper 4.9.0?
I would like to introduce a notch for cylinder by using option _cut_. My question is that: Is it possible to regenerate the cells and control the number of cells after cutting the notch?
Thanks a lot for your help!

**Comment by rquey** [comment, marked as answer]
No, it is not. The cutting operation generally remove cells and it is not possible to know in advance the number of cells after cutting.  So, for now, what you have to do is to increase the initial number of cells to compensate.
